src/platform/packages/shared/kbn-unified-metrics-grid/src/components/empty_state/empty_state.tsx

```diff
@@ -68,11 +69,17 @@ export const EmptyState = ({ isLoading = false }: EmptyStateProps) => (
   </div>
 );
 
-export const ChartLoadingProgress = () => (
+export const GridLoadingProgress = () => (
   <EuiProgress
     size="xs"
     color="accent"
     position="absolute"
     data-test-subj="metricsExperienceProgressBar"
   />
 );
+
+export const ChartLoadingIcon = () => (
```

Comment:
NIT: Do we plan to reuse this? IMO if it's used only once it can stay inline, wdyt?